label_sync/main.go

```diff
@@ -688,39 +690,32 @@ func main() {
 			logrus.WithError(err).Fatal("failed to create client")
 		}
 
-		var filt filter
-		switch {
-		case *onlyRepos != "":
-			if *skipRepos != "" {
-				logrus.Fatalf("--only and --skip cannot both be set")
-			}
-			only := make(map[string]bool)
-			for _, r := range strings.Split(*onlyRepos, ",") {
-				only[strings.TrimSpace(r)] = true
-			}
-			filt = func(org, repo string) bool {
-				_, ok := only[org+"/"+repo]
-				return ok
-			}
-		case *skipRepos != "":
-			skip := make(map[string]bool)
-			for _, r := range strings.Split(*skipRepos, ",") {
-				skip[strings.TrimSpace(r)] = true
-			}
-			filt = func(org, repo string) bool {
-				_, ok := skip[org+"/"+repo]
-				return !ok
-			}
-		default:
-			filt = func(o, r string) bool {
-				return true
-			}
-		}
-
 		for _, org := range strings.Split(*orgs, ",") {
 			org = strings.TrimSpace(org)
+			logger := logrus.WithField("org", org)
 
-			if err = syncOrg(org, githubClient, *config, filt); err != nil {
+			var repos []string
+			switch {
+			case *onlyRepos != "":
```

Comment:
The docs say that onlyRepos and skipRepos should be of the form `kubernetes/test-infra`, not `test-infra`. 

It looks like this is changing things so that its now the reverse (flag has to be `test-infra` not `kubernetes/test-infra`). If so then please update the flag docs.

Personally I prefer the less ambiguous `kubernetes/test-infra`. If the value is `test-infra` there is no way to say "only skip kubernetes/test-infra but please process kubernetes-sigs/test-infra`. 

If we do keep the org name, then it doesn't seem like we need the `for _, org := range *orgs {` loop above since this is all contained by the onlyRepos flag.

Reply:
Ah, crap -- I want to keep `org/repo`. I will update

Reply:
Maybe add a test for this behavior? :)